src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -50,6 +50,7 @@
 import net.minecraftforge.items.IItemHandlerModifiable;
 
 import javax.annotation.Nullable;
+
```

Comment:
Please don't do unnecessary changes to files which are not directly related to given changes.

I know you were touching this file in previous commits but right now there is no to touch this file now.